README.md

```diff
@@ -158,7 +158,13 @@ Update the `CFBundleLocalizations` array in the `Info.plist` at `ios/Runner/Info
     }
 }
 ```
+### Generating the translations
 
+In order to use the translations, you need to generate them. From the root folder of the app, run the following command:
+
+```
+flutter gen-l10n
+```
```

Comment:
```suggestion
To use the latest translations changes, you will need to generate them beforehand. Flutter will automatically generate them per `flutter run`. Alternatively, if you would like to avoid running the entire application, you may do:

```sh
# 💬 Generate the newest l10n files (from project root):
flutter gen-l10n
```
```